MockKit/MockCGMManager.swift

```diff
@@ -21,6 +21,18 @@ public struct MockCGMState: SensorDisplayable {
     
     public var glucoseValueType: GlucoseValueType?
     
+    private var urgentLowGlucoseThresholdValue: Double
```

Comment:
[nit] Consider adding mg/dL indication this this variable name since you have hard-coded these units in the associated computed property.

Reply:
Or, consider including a "unit" property that could default to mg/dL and use that property in the various computed properties below.

Reply:
adding `unit`